### PR TITLE
API basics and RO /api/projects endpoint

### PR DESCRIPTION
**Pull request by simi**
screenshot-localhost 3000 2014-12-02 22-28-19

### CODE REVIEW

**Comment by papricek** [comment]
Would be great to finish this one. You can easily get some help from @Cervajz

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -34,6 +34,11 @@ def update_settings
     end
   end
 
+  def reset_token
+    current_user.generate_token
+    redirect_to settings_path, :notice => t('users.token_reseted')
```
Comment (Cervajz) [inline app/controllers/users_controller.rb]: reseted => reset

File: config/locales/application.cs.yml [hunk 2]
```diff
@@ -66,6 +66,9 @@ cs:
     update: "Aktualizovat"
     salary: "Mzda"
     user: "Uživatel"
+    token: "Token"
+    token_reseted: "Token vyresetován."
```
Comment (Cervajz) [inline config/locales/application.cs.yml]: reseted => reset

File: config/locales/application.en.yml [hunk 3]
```diff
@@ -67,6 +67,9 @@ en:
     update: "Update"
     salary: "Salary"
     user: "User"
+    token: "Token"
+    token_reseted: "Token reseted."
```
Comment (Cervajz) [inline config/locales/application.en.yml]: Token has been reset.

**Pull request closed without merging**